gherkin/java/src/main/java/io/cucumber/gherkin/GherkinDialectProvider.java

```diff
@@ -44,18 +24,16 @@ public GherkinDialect getDefaultDialect() {
 
     public GherkinDialect getDialect(String language, Location location) {
         requireNonNull(language);
+        GherkinDialect dialect = GherkinDialects.DIALECTS.get(language);
 
-        JsonValue languageObject = DIALECTS.get(language);
-        if (languageObject == null) {
+        if (dialect == null) {
             throw new ParserException.NoSuchLanguageException(language, location);
         }
 
-        return new GherkinDialect(language, languageObject.asObject());
+        return dialect;
     }
 
     public List<String> getLanguages() {
-        List<String> languages = new ArrayList<>(DIALECTS.names());
-        sort(languages);
-        return unmodifiableList(languages);
+        return unmodifiableList(asList("en", "no"));
```

Comment:
Should this be `GherkinDialects.DIALECTS.keySet()`?

Reply:
And if we're breaking things, may well change the return type to a set.

Reply:
Ah yes - leftover from the first increment